docs/xxmergecompileroptions.md

```diff
@@ -36,11 +36,19 @@ This option enables or disables the merging of multiple `-Xjit` or `-Xaot` optio
 
 ## Explanation
 
- By default, if you specify `-Xjit` option (or `-Xaot` option) multiple times, only the last option takes effect. Therefore, on adding an `-Xjit` option, the resultant behavior might not be as expected, if any instance of this option already exists. It is time-consuming to find the existing instances and try combining them manually so that the system behaves as required.
+By default, if you specify `-Xjit` option (or `-Xaot` option) multiple times, only the last option takes effect. Therefore, on adding an `-Xjit` option, the resultant behavior might not be as expected, if any instance of this option already exists. It is time-consuming to find the existing instances and try combining them manually so that the system behaves as required.
 
- You can use the `-XX:+MergeCompilerOptions` option to merge all the existing instances of the `-Xjit` options logically. To retain the default behavior, whereby only the last option takes effect, you can use the `-XX:-MergeCompilerOptions` option.
+You can use the `-XX:+MergeCompilerOptions` option to merge all the existing instances of the `-Xjit` options logically. To retain the default behavior, whereby only the last option takes effect, you can use the `-XX:-MergeCompilerOptions` option.
 
- If both `-Xjit` and `-Xaot` options exist and you use the `-XX:+MergeCompilerOptions` option, then multiple `-Xjit` options are merged into a single `-Xjit` option and multiple `-Xaot` options are merged into a single `-Xaot` option.
+If both `-Xjit` and `-Xaot` options exist and you use the `-XX:+MergeCompilerOptions` option, then multiple `-Xjit` options are merged into a single `-Xjit` option and multiple `-Xaot` options are merged into a single `-Xaot` option.
```

Comment:
I think that the word "both" can be interpreted wrongly. You don't need to actually have both -Xjit **and** -Xaot present on the command line. Maybe it can rephrased like this:
If  multiple `-Xjit` options are present on the command line and you use the `-XX:+MergeCompilerOptions` option, then these multiple `-Xjit` options are merged into a single `-Xjit` option.
Similarly, if  multiple `-Xaot` options are present on the command line and you use the `-XX:+MergeCompilerOptions` option, then these multiple `-Xaot` options are merged into a single `-Xaot` option.

Reply:
I was of the understanding that this statement was explaining the scenario where **both** `-Xjit` and `-Xaot` options exist in the command line then in such a scenario if you use the `-XX:+MergeCompilerOptions` option, then multiple `-Xjit` options are merged into a single `-Xjit` option and multiple `-Xaot` options are merged into a single `-Xaot` option.

Is there any other possible interpretation?

In the previous paragraph, we have explained the merging concept 

> You can use the `-XX:+MergeCompilerOptions` option to merge all the existing instances of the `-Xjit` options logically.

Reply:
ok, I'm fine with the original description. It leaves unspecified the case where you only have -Xaot options; maybe that can be inferred.

Reply:
Maybe I can add (or `-Xaot` options) to the sentence in the previous para?

> You can use the -XX:+MergeCompilerOptions option to merge all the existing instances of the -Xjit options (or `-Xaot` options) logically.

Reply:
Yes, that will do.